benchmarks/cpp/nvfuser/bert.cpp

```diff
@@ -342,11 +342,6 @@ static void MagicScheduler_BiasDropoutAddLayernormFwd(
       bytes * int64_t(benchmark_state.iterations()));
 }
 
```

Comment:
@zasdfgbnm do you want to keep these changes or was this just for testing?

Reply:
I think we can keep? The code for `BiasDropoutAddLayernormFwd` was there but never executed. Looks like someone wrote it but forget to enable it.